fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/LoanRepository.java

```diff
@@ -163,4 +165,7 @@ List<Loan> findByGroupOfficeIdsAndLoanStatus(@Param("officeIds") Collection<Long
 
     boolean existsByExternalId(@Param("externalId") String externalId);
 
+    @Query(FIND_ALL_NON_CLOSED)
+    List<Integer> findAllNonClosedLoan();
```

Comment:
I'd rename this method to `findAllNonClosedLoanIds` since we're returning a list of IDs, not a list of Loans.